src/styles/main.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
```

Comment:
file name should match BEM block name - `catalog.scss`

Reply:
I understand